packages/opal-client/opal_client/data/fetcher.py

```diff
@@ -14,7 +14,14 @@
 class DataFetcher:
     """fetches policy data from backend."""
 
-    def __init__(self, default_data_url: str = None, token: str = None):
+    # Use as default config the configuration provider by opal_client_config.DATA_STORE_CONN_RETRY
+    # Add reraise as true (an option not available for control from the higher-level config)
+    DEFAULT_RETRY_CONFIG = opal_client_config.DATA_STORE_CONN_RETRY.toTenacityConfig()
+    DEFAULT_RETRY_CONFIG["reraise"] = True
```

Comment:
it's to raise the original exception encountered after the last retry failure instead of it being part of the `RetryError` of tenacity, i have basically kept it same as `POLICY_UPDATER_CONN_RETRY`  under `PolicyFetcher`

i am not sure if making it configurable helps as i would mostly want to see the original exception raised instead of if with in the `RetryError` from tenacity

Reply:
OK, sounds good